warehouse/templates/manage/organization/settings.html

```diff
@@ -28,6 +28,12 @@ <h2>{% trans %}Organization settings{% endtrans %}</h2>
     <div class="form-group">
       <span class="form-group__label">{% trans %}Organization account name{% endtrans %}</span>
       <p class="form-group__text">{{ organization.name }}</p>
+      <p class="form-group__help-text">
+        {% trans %}This account name is used in URLs on PyPI.{% endtrans %}<br>
+        {% trans %}Change the organization account name <a href="#change-organization-account-name">below</a>.{% endtrans %}
```

Comment:
Suggestion: extract url to a variable, per translation docs guidelines 
https://warehouse.pypa.io/translations.html#passing-non-translatable-values-to-translated-strings

Reply:
+1